FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Mauro Worobiej on 05/09/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+class FeedImageMapper {
```

Comment:
This class is not intended to be inherited thus it should be `final`.
```suggestion
final class FeedImageMapper {
```

Reply:
Make sense, thanks!